src/types/k8s.go

```diff
@@ -6,9 +6,35 @@ type ZarfState struct {
 	Distro        string       `json:"distro"`
 	Architecture  string       `json:"architecture"`
 	StorageClass  string       `json:"storageClass"`
-	Secret        string       `json:"secret"`
-	NodePort      string       `json:"nodePort"`
 	AgentTLS      GeneratedPKI `json:"agentTLS"`
+
+	GitServer             GitServerInfo         `json:"gitServer"`
+	ContainerRegistryInfo ContainerRegistryInfo `json:"containerRegistryInfo"`
+	LoggingSecret         string                `json:"loggingSecret"`
+}
+
+type GitServerInfo struct {
+	PushUsername string `json:"pushUsername"`
```

Comment:
It probably doesn't matter as much since this isn't exposed in the `zarf.yaml`, but I've been trying to fill out at least the jsonschema description fields in the types pkg.